packages/wrangler/src/config/environment.ts

```diff
@@ -1,3 +1,4 @@
+import type { CreateApplicationRequest } from "../cloudchamber/client";
```

Comment:
Types in this file are used to generate the public JSON schema for Wrangler's config, so ideally we want to keep the dependencies minimal. I know it's not ideal, but would you be able to duplicate the relevant `ContainerApp` type in this file, along with doc comments?

Reply:
There is like a huge amount of types we have to copy for this... And this type specifically gets new added properties and change. Are we sure we can't get away with importing this type?

Reply:
If possible, writing it out in this file would be much better. It would also allow for handwriting doc comments to explain what each property does. This forms part of Wrangler's public config contract—changing based on changes to autogenerated code doesn't give much oversight over what's ending up in the config file.

Having said that, I wonder if the fact that this type is super detailed is in itself a bit of a red flag? What types of things does it contain? Why is it so big?